scintilla/win32/ScintillaWin.cxx

```diff
@@ -1463,7 +1463,11 @@ sptr_t ScintillaWin::WndProc(unsigned int iMessage, uptr_t wParam, sptr_t lParam
 						} else if (PointIsHotspot(PointFromPOINT(pt))) {
 							DisplayCursor(Window::cursorHand);
 						} else {
-							DisplayCursor(Window::cursorText);
+							Sci::Position pos = PositionFromLocation(PointFromPOINT(pt), true, true);

```

Comment:
One more stuff, in order to make future Scintilla updating easier, could you add the following comment:
// Fix the following issue:
// https://github.com/notepad-plus-plus/notepad-plus-plus/issues/8588
// It should be fixed in Scintilla release superior to v4.4.4 - to verify.